<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Sheet Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="copyCountSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>165</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="copyCountLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>135</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copies:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>480</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>480</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateYearLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>75</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateMonthLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>15</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Month:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>455</width>
      <height>450</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="monthComboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>45</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>105</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>2018</number>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>480</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
